src/meta_schedule/task_scheduler/task_scheduler.cc

```diff
@@ -94,6 +94,8 @@ void SendToRunner(const Runner& runner, const TuneContext& context, PackedFunc l
 
 void TaskSchedulerNode::InitializeTask(int task_id) {
   TuneContext task = this->tasks[task_id];
+  TVM_PY_LOG(INFO, this->logging_func)
```

Comment:
remove it?

Reply:
this is expected behavior that we print something out on screen